domains/ali-samir.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TheThreeL",
+        "email": "alisamir3384408@gmail.com"
+    },
+    "record": {
+        "CNAME": "2fa83007-d49e-4418-83a2-536205c253aa-00-tpnws5m92s2x.worf.replit.dev"
```

Comment:
I guess you turned off your repl, replit says
"The Repl is currently not running."

Reply:
Should the repil be always open?
If yes, try opening it again